docs/conditional-steps.md

```diff
@@ -1,29 +1,26 @@
-# Conditional Steps
+# Conditional steps
 
 Conditional steps allow the definition of steps that only run if a condition is
-met. For example, an orb could define a command that runs a set of steps if the
+met. For example, an orb could define a command that runs a set of steps *if* the
 orb's user invokes it with `myorb/foo: { dostuff: true }`, but not
 `myorb/foo: { dostuff: false }`.
 
-Note that the condition is checked before a workflow is actually run. This
+These conditions are checked before a workflow is actually run. That
```

Comment:
These docs can't seem to decide if they want to be 80 char wide or full-line width. :thinking:

Reply:
the curse of many authors. Personally, I like soft wrap for markdown text.